test/e2e/framework/kubesystem/kubesystem.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubesystem
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"time"
+
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2essh "k8s.io/kubernetes/test/e2e/framework/ssh"
+)
+
+const (
+	// ssh port
+	sshPort = "22"
```

Comment:
There are several `sshPort` const in e2e sub framework. I think we may use a common one in e2e core framework?

Reply:
That is a nice point.
But that should not be in this PR as the scope.

Reply:
Sure.